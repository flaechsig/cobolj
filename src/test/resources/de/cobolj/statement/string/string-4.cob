IDENTIFICATION DIVISION. 
    PROGRAM-ID. HELLO-WORLD.
    
DATA DIVISION.
WORKING-STORAGE SECTION.
  01 WS-RESULT PIC A(15).
  01 WS-POINTER PIC 9(2) VALUE 3.
  
PROCEDURE DIVISION.
	STRING 'HELLO' ' ' 'WORLD' DELIMITED SIZE 
	    INTO WS-RESULT
	    WITH POINTER WS-POINTER
	    ON OVERFLOW 
	        DISPLAY 'OVERFLOW ERROR'
	        DISPLAY 'SECOND LINE'
	    NOT ON OVERFLOW 
	        DISPLAY 'NO OVERFLOW ERROR'
	END-STRING.
    DISPLAY WS-RESULT.
    DISPLAY WS-POINTER.
    