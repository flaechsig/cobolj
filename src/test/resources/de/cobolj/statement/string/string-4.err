Fehler beim Umgang mit der Pointer-Phrase