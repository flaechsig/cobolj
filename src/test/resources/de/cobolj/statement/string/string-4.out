NO OVERFLOW ERROR
  HELLO WORLD  
14
